gtk3/sample/misc/tooltips.rb

```diff
@@ -5,28 +5,27 @@
 
   Copyright (c) 2007-2015 Ruby-GNOME2 Project Team
   This program is licenced under the same licence as Ruby-GNOME2.
-
-  $Id: tooltips.rb,v 1.1 2007/07/10 13:17:34 ggc Exp $
 =end
 
-require 'gtk3'
+require "gtk3"
 
-unless Gtk::Version.or_later?(2, 12, 0)
-    puts "This sample requires GTK+ 2.12.0 or later"
-    exit
+unless Gtk::Version.or_later?(3, 4, 2)
+  puts "This sample requires GTK+ 3.4.2 or later: #{Gtk::Version::STRING}"
+  exit
 end
 
+
 def treeview_query_tooltip(treeview, keyboard_tip, x, y, tooltip)
     if keyboard_tip
         # Keyboard mode
-        path, = treeview.cursor
+        ok,path = treeview.cursor
         if not path
             return false
         end
     else
         bin_x, bin_y = treeview.convert_widget_to_bin_window_coords(x, y)
         # Mouse mode
-        path, = treeview.get_path_at_pos(bin_x, bin_y)
+        ok, path = treeview.get_path_at_pos(bin_x, bin_y)
```

Comment:
It should be fixed in library. I fixed it. Now `#get_path_at_pos` returns `[Gtk::TreePath, Gtk::TreeViewColumn, cell_x, cell_y]` if path is found, `nil` otherwise.

Could you update the code?